simulation_rate: 250
clock_rate: 250
realtime_factor: 1.0

iterate_without_input: true

individual_takeoff_platform:
  enabled: false

uav_names: [
  "uav1",
]

uav1:
  type: "x500"
  spawn:
    x: 0.0 #10.0
    y: 0.0 #15.0
    z: 2.0
    heading: 0
